.circleci/windows/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
This script was extracted from build.bat (which was simply running bash commands)